openadapt/utils.py

```diff
@@ -310,13 +454,31 @@ def get_scale_ratios(action_event):
 
 
 def display_event(
-    action_event,
-    marker_width_pct=0.03,
-    marker_height_pct=0.03,
-    marker_fill_transparency=0.25,
-    marker_outline_transparency=0.5,
-    diff=False,
-):
+    action_event: Any,
```

Comment:
Please replace `Any` with `models.ActionEvent`

Reply:
resolved in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/4d079dc4023313ab796a1398cd95506a1885013c